RELEASE.md

```diff
@@ -0,0 +1,39 @@
+# Releasing p2panda
+
+_This example assumes we are publising version `1.2.0`._
+
+_Remember to update the version number accordingly!_
+
+## Checks and preparations
+
+1. Check that the CI has passed on the p2panda project's
+  [Github page](https://github.com/p2panda/p2panda).
+2. Make sure you are on the `main` branch.
+3. Run the test suite and make sure all tests pass:
+    - `cargo test --all-features`
+4. Make sure that all examples in each crate `README.md` (including the ones in the
```

Comment:
This seems to be a thing: https://github.com/p2panda/p2panda/issues/541 but nothing important for this PR, just FYI